Unable to compile site:

```
[0]  warning  in ./node_modules/component-classes/index.js
[0] 
[0] Module not found: Error: Can't resolve 'indexof' in '/home/vsts/work/1/s/node_modules/component-classes'
[0] 
[0] (node:3083) UnhandledPromiseRejectionWarning: TypeError: ssr is not a function
[0]     at /home/vsts/work/1/s/node_modules/bisheng/lib/index.js:319:11
[0]     at new Promise (<anonymous>)
[0]     at /home/vsts/work/1/s/node_modules/bisheng/lib/index.js:317:16
[0]     at Array.map (<anonymous>)
[0]     at /home/vsts/work/1/s/node_modules/bisheng/lib/index.js:314:50
[0]     at finalCallback (/home/vsts/work/1/s/node_modules/webpack/lib/Compiler.js:257:39)
[0]     at /home/vsts/work/1/s/node_modules/webpack/lib/Compiler.js:306:14
[0]     at AsyncSeriesHook.eval [as callAsync] (eval at create (/home/vsts/work/1/s/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:42:1)
[0]     at AsyncSeriesHook.lazyCompileHook (/home/vsts/work/1/s/node_modules/tapable/lib/Hook.js:154:20)
[0]     at /home/vsts/work/1/s/node_modules/webpack/lib/Compiler.js:304:22
[0]     at Compiler.emitRecords (/home/vsts/work/1/s/node_modules/webpack/lib/Compiler.js:499:39)
[0]     at /home/vsts/work/1/s/node_modules/webpack/lib/Compiler.js:298:10
[0]     at /home/vsts/work/1/s/node_modules/webpack/lib/Compiler.js:485:14
[0]     at eval (eval at create (/home/vsts/work/1/s/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:18:1)
[0] (node:3083) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 256)
[0] (node:3083) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
[0]     at emitDeprecationWarning (internal/process/promises.js:153:11)
[0]     at processPromiseRejections (internal/process/promises.js:205:13)
[0]     at processTicksAndRejections (internal/process/task_queues.js:94:32)
[0] bisheng build --ssr -c ./site/bisheng.config.js exited with code 0
```

https://dev.azure.com/ant-design/ant-design/_build/results?buildId=7741&view=logs&j=a2b3994b-89d0-550c-2130-d92890401028&t=b04045b3-20f4-593c-3009-87be7f4e3abb

--- a/package.json
+++ b/package.json
@@ -258,6 +258,7 @@
     "theme-switcher": "^1.0.2",
     "typescript": "~3.8.2",
     "webpack-bundle-analyzer": "^3.6.0",
+    "webpack-plugin-esbuild": "^1.0.0-beta.2",
     "xhr-mock": "^2.4.1",
     "xhr2": "^0.2.0",
     "yaml-front-matter": "^4.0.0"

--- a/package.json
+++ b/package.json
@@ -181,6 +181,7 @@
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.14.0",
     "enzyme-to-json": "^3.3.5",
+    "esbuild-webpack-plugin": "^1.0.0-beta.3",
     "eslint": "^6.8.0",
     "eslint-config-airbnb": "^18.0.0",
     "eslint-config-prettier": "^6.0.0",
@@ -258,7 +259,6 @@
     "theme-switcher": "^1.0.2",
     "typescript": "~3.8.2",
     "webpack-bundle-analyzer": "^3.6.0",
-    "webpack-plugin-esbuild": "^1.0.0-beta.2",
     "xhr-mock": "^2.4.1",
     "xhr2": "^0.2.0",
     "yaml-front-matter": "^4.0.0"

--- a/site/bisheng.config.js
+++ b/site/bisheng.config.js
@@ -1,7 +1,7 @@
 const path = require('path');
 const replaceLib = require('@ant-design/tools/lib/replaceLib');
 const getWebpackConfig = require('@ant-design/tools/lib/getWebpackConfig');
-const EsbuildPlugin = require('webpack-plugin-esbuild').default;
+const EsbuildPlugin = require('esbuild-webpack-plugin').default;
 const { version } = require('../package.json');
 
 const { webpack } = getWebpackConfig;

--- a/site/bisheng.config.js
+++ b/site/bisheng.config.js
@@ -1,6 +1,7 @@
 const path = require('path');
 const replaceLib = require('@ant-design/tools/lib/replaceLib');
 const getWebpackConfig = require('@ant-design/tools/lib/getWebpackConfig');
+const EsbuildPlugin = require('webpack-plugin-esbuild').default;
 const { version } = require('../package.json');
 
 const { webpack } = getWebpackConfig;
@@ -134,6 +135,8 @@ module.exports = {
       // Resolve use react hook fail when yarn link or npm link
       // https://github.com/webpack/webpack/issues/8607#issuecomment-453068938
       config.resolve.alias = { ...config.resolve.alias, react: require.resolve('react') };
+    } else {
+      config.optimization.minimizer = [new EsbuildPlugin()];
     }
 
     alertBabelConfig(config.module.rules);

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -3,7 +3,7 @@
 const getWebpackConfig = require('@ant-design/tools/lib/getWebpackConfig');
 const IgnoreEmitPlugin = require('ignore-emit-webpack-plugin');
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
-const EsbuildPlugin = require('webpack-plugin-esbuild').default;
+const EsbuildPlugin = require('esbuild-webpack-plugin').default;
 const darkVars = require('./scripts/dark-vars');
 const compactVars = require('./scripts/compact-vars');
 

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -36,15 +36,6 @@ function externalMoment(config) {
 
 function processWebpackThemeConfig(themeConfig, theme, vars) {
   themeConfig.forEach(config => {
-    if (process.env.RUN_ENV === 'PRODUCTION') {
-      ignoreMomentLocale(config);
-      externalMoment(config);
-      // Reduce non-minified dist files size
-      config.optimization.usedExports = true;
-      // use esbuild
-      config.optimization.minimizer[0] = new EsbuildPlugin();
-    }
-
     // rename default entry to ${theme} entry
     Object.keys(config.entry).forEach(entryName => {
       config.entry[entryName.replace('antd', `antd.${theme}`)] = config.entry[entryName];

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -3,6 +3,7 @@
 const getWebpackConfig = require('@ant-design/tools/lib/getWebpackConfig');
 const IgnoreEmitPlugin = require('ignore-emit-webpack-plugin');
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+const EsbuildPlugin = require('webpack-plugin-esbuild').default;
 const darkVars = require('./scripts/dark-vars');
 const compactVars = require('./scripts/compact-vars');
 
@@ -35,8 +36,14 @@ function externalMoment(config) {
 
 function processWebpackThemeConfig(themeConfig, theme, vars) {
   themeConfig.forEach(config => {
-    ignoreMomentLocale(config);
-    externalMoment(config);
+    if (process.env.RUN_ENV === 'PRODUCTION') {
+      ignoreMomentLocale(config);
+      externalMoment(config);
+      // Reduce non-minified dist files size
+      config.optimization.usedExports = true;
+      // use esbuild
+      config.optimization.minimizer[0] = new EsbuildPlugin();
+    }
 
     // rename default entry to ${theme} entry
     Object.keys(config.entry).forEach(entryName => {
@@ -61,6 +68,7 @@ function processWebpackThemeConfig(themeConfig, theme, vars) {
 const webpackConfig = getWebpackConfig(false);
 const webpackDarkConfig = getWebpackConfig(false);
 const webpackCompactConfig = getWebpackConfig(false);
+
 if (process.env.RUN_ENV === 'PRODUCTION') {
   webpackConfig.forEach(config => {
     ignoreMomentLocale(config);
@@ -68,6 +76,8 @@ if (process.env.RUN_ENV === 'PRODUCTION') {
     addLocales(config);
     // Reduce non-minified dist files size
     config.optimization.usedExports = true;
+    // use esbuild
+    config.optimization.minimizer[0] = new EsbuildPlugin();
     // skip codesandbox ci
     if (!process.env.CSB_REPO) {
       config.plugins.push(

